### PR TITLE
add filament-users & filament-translations plugin and my author profile

### PR DESCRIPTION
**Pull request by 3x1io**

### CODE REVIEW

File: content/plugins/filament-translations.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+name: Filament Users
```
Comment (saade) [inline content/plugins/filament-translations.md:2]:
```suggestion
name: Translations
```

File: content/plugins/filament-translations.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+slug: filament-translations
```
Comment (saade) [inline content/plugins/filament-translations.md:3]:
```suggestion
slug: fadymondy-translations
```

File: content/plugins/filament-users.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+---
+name: Filament Users
```
Comment (saade) [inline content/plugins/filament-users.md:2]:
```suggestion
name: Users
```

File: content/plugins/filament-users.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+slug: filament-users
```
Comment (saade) [inline content/plugins/filament-users.md:3]:
```suggestion
slug: fadymondy-users
```

**Review by saade — CHANGES_REQUESTED** [review]
Please do not include the word "Filament" in the name or slug of your plugin, or the file names. It is redundant, as all plugins on the website are for Filament.

**Comment by 3x1io** [comment]
Oky i will on the next plugins

**Comment by saade** [comment]
> Oky i will on the next plugins

this plugin will not be accepted if it does not follow the [plugin guidelines](https://github.com/filamentphp/filamentphp.com#submitting-a-plugin). You don't need to accept exacly my suggestions, just make sure to not include the word `Filament` in your plugin name or slug.

**Comment by 3x1io** [comment]
np i will fixit and resubmit, thanks

**Comment by danharrin** [comment]
Please reopen when you have fixed it.

**Pull request closed without merging**